src/pal/src/shmemory/shmemory.cpp

```diff
@@ -455,14 +455,14 @@ BOOL SHMInitialize(void)
     init_waste();
     
 #ifndef CORECLR
```

Comment:
We can remove all the code inside #ifndef CORECLR from PAL. @jkotas I hope I am not missing something.